Assets/Prefabs/GunParts/CoilBarrel.prefab

```diff
@@ -411,9 +411,14 @@ MonoBehaviour:
     addition: 1.2
     multiplier: 0
     exponential: 1
+  - name: CriticalMultiplier
+    addition: 0.3
+    multiplier: 0
+    exponential: 1
```

Comment:
I think crits are best when they are a directly because the player took a risk to aim at a small area in order to get more reward (damage). I think the random nature of the coil makes it a bad candidate for headshot crits

Reply:
I also think the Telescope could "grant" barrels that don't normaly have headshot crits the ability, like the coil, to do so

Reply:
Rebalanced accordingly.